install_config/persistent_storage/topics/glusterfs_using_storage.adoc

```diff
@@ -9,19 +9,19 @@ NGINX pod.
 apiVersion: v1
 kind: Pod
 metadata:
-  name: nginx-pod
+  name: hello-openshift-pod
   labels:
-    name: nginx-pod
+    name: hello-openshift-pod
 spec:
   containers:
-  - name: nginx-pod
-    image: nginx
+  - name: hello-openshift-pod
+    image: openshift/hello-openshift
     ports:
     - name: web
       containerPort: 80
     volumeMounts:
     - name: gluster-vol1
-      mountPath: /usr/share/nginx/html
+      mountPath: /usr/share/hello-openshift/html
```

Comment:
this volumeMount doesn't make sense for `hello-openshift`.  might leave this one as-is since the volumeMount is needed to demonstrate using the gluster volume.